docs/use/onboarding/index.md

```diff
@@ -2,125 +2,175 @@
 
-Tinlake pools are backed by real-world assets such as invoices, residential real-estate loans or trade receivables. Financing those assets on Tinlake requires a legal structure that give the investors a legal claim on the assets. To ensure this recourse, each Asset Originator sets up a legal structure commonly used in the traditional financial system - a “special purpose vehicle” ("SPV"). The collateral for the individual assets are assigned to this legal entity to ensure investors have a legal claim to the underlying assets and the assets are held independently ("banktruptcy remote") from the Asset Originator.
+![Steps to onboard as investor and onboard to pool](./images/Onboarding overview.png)
 
-## Why onboarding and KYC
+**Onboard as investor**: Onboarding as an investor is a one-time step to verify your profile for KYC ("Know your customer") through the Securitize service. Once onboarded as an investor you are eligible to invest in any Tinlake pool.
 
-With such an underlying legal structure, investments in Tinlake Pool's usually are private placements from a regulatory perspective. This comes with certain regulatory requirements For example, behind many Tinlake pools are SPVs incorporated in Delaware. Private placements from these SPVs are conducted under `Reg D` or `Reg S` of the [`US Securities Act of 1933`](https://www.govinfo.gov/content/pkg/COMPS-1884/pdf/COMPS-1884.pdf) which means US investors must be accredited and all investors must clear standard KYC ("Know Your Customer") checks. All investors must also sign a subscription agreement ("SubDoc") with the pools "Issuer", so the legal entity/SPV behind the pool.
+**Onboard to pool**: Onboarding to a pool requires signing a subscription agreement with the pool issuer. Once the agreement has been signed by both parties, you are automatically whitelisted for investment.
 
-This legal set-up leads to certain restrictions. For example, there usually is a minimum investment amount per pool, US investors have to be `Accredited Investors` and, unfortunately, investors from certain countries may be excluded due to sanctions or missing tax treaties.
+### Requirements
+**Onboarding as an investor** requires:
+- Ethereum wallet, e.g. Metamask
+- A passport or government-issued photo ID
+- Your tax number
+- Proof of address (e.g. utility bill, phone bill or council tax bill, not older than 3 months)
 
-This obviously contradicts the idea of decentralized and open finance and the Centrifuge team is working hard to allow more people to invest within the legal framework given by regulators.
+**Onboarding to a pool** requires:
+- Ethereum wallet
+- A verified KYC in Securitize (i.e. a completed onboarding as an investor)
+- A minimum investment amount of 5,000 DAI
 
-## Onboarding quick guide
+#### Further requirements
+- **US investors** need to be "Acreddited investors" to invest in Tinlake pools. *See section [Onboarding as US investor](#Onboarding-as-an-US-investor)*
```

Comment:
@onnovisser Could you help us out and double check that this is the correct syntax for referencing to a section within this document? Not working properly for me on the preview

Reply:
is this used anywhere else? some features of markdown might not be supported out-of-the-box. we might need to add a plugin to make this work